packages/modules/vehicles/vwid/soc.py

```diff
@@ -24,12 +24,16 @@ def __init__(self, device_config: Union[dict, VWId], vehicle: int):
         self.store = store.get_car_value_store(self.vehicle)
         self.component_info = ComponentInfo(self.vehicle, self.config.name, "vehicle")
 
-    def update(self, soc_update_data: SocUpdateData) -> None:
+    def update(self, charge_state: bool = False) -> None:
```

Comment:
```suggestion
    def update(self, soc_update_data: SocUpdateData) -> None:
```
Das war richtig. update_soc.py übergibt ein SocUpdateData-Objekt.